packages/xy-chart/src/series/LineSeries.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { curveCardinal, curveLinear } from '@vx/curve';
+import { GlyphDot } from '@vx/glyph';
+import { LinePath } from '@vx/shape';
+
+import { callOrValue } from '../utils/chartUtils';
+import { colors } from '../theme';
+import { lineSeriesDataShape } from '../utils/propShapes';
+
+const propTypes = {
+  data: lineSeriesDataShape.isRequired,
+  interpolation: PropTypes.oneOf(['linear', 'cardinal']), // @todo add more
+  label: PropTypes.string.isRequired,
+  showPoints: PropTypes.bool,
+
+  stroke: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  strokeDasharray: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  strokeWidth: PropTypes.oneOfType([PropTypes.func, PropTypes.number]),
+
+  // these will likely be injected by the parent chart
+  xScale: PropTypes.func.isRequired,
+  yScale: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  interpolation: 'cardinal',
+  showPoints: false,
+  stroke: colors.default,
+  strokeDasharray: null,
+  strokeWidth: 3,
+};
+
+export default function LineSeries({
+  data,
+  interpolation,
+  label,
+  showPoints,
+  stroke,
+  strokeDasharray,
+  strokeWidth,
+  xScale,
+  yScale,
+}) {
+  const x = d => d.x;
+  const y = d => d.y;
```

Comment:
you could define these outside the `render` function so you don't have to create the function every render... probably a premature optimization but just leaving a note.